test/testdata/dbbuilder/inbox/builder.go

```diff
@@ -41,8 +41,8 @@ func (b Builder) CreateMessage(contentType inbox.ContentType, contentID, postedB
 func (b Builder) CreateUserInboxMessage(userID, messageID uuid.UUID) []inbox.UserInboxMessage {
```

Comment:
Since the `CreateUserInboxMessage` method always returns only one Message, consider changing the return type from an array to a single Message.